aml_module/invoker.py

```diff
@@ -36,67 +34,92 @@ def is_timestamp_ascending(timestamps):
     return 0
 
 
-def invoke(input_path, detect_mode, timestamp_column, value_column, batch_size, threshold, sensitivity,
-            appendMode, compute_stats_in_visualization, output_path):
-    data_frame_directory = load_data_frame_from_directory(input_path)
+def read_as_dataframe(input_path: str):
+    if os.path.isfile(input_path):
+        if input_path.endswith(".csv"):
+            return pd.read_csv(input_path)
+        elif input_path.endswith(".parquet"):
```

Comment:
Why does parquet a data type that we want to support?

Reply:
We just want to support the case that when data type is parquet, the module can process it properly.

Reply:
As there are scenarios with this kind of data source, just keep these codes.